bluebottle/funding/tests/test_admin.py

```diff
@@ -153,6 +154,18 @@ def test_donation_reward(self):
         self.assertTrue(second.title in response.content.decode('utf-8'))
         self.assertFalse(third.title in response.content.decode('utf-8'))
 
+    def test_contribution_date(self):
+        donation = DonorFactory.create(
+            activity=self.funding,
+            amount=Money(100, 'EUR')
+        )
+        donation.states.succeed(save=True)
+        self.client.force_login(self.superuser)
+        response = self.client.get(self.admin_url)
+        curr_time = now()
+        reported_time = f'{defaultfilters.date(curr_time)}, {defaultfilters.time(curr_time)}'
+        self.assertTrue(reported_time in response.content.decode('utf-8'))
```

Comment:
Better to explicitly check that this value appears in the first column of the table.